lib/rules/padding-lines-in-component-definition.js

```diff
@@ -0,0 +1,357 @@
+/**
+ * @author ItMaga <https://github.com/ItMaga>
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+/**
+ * @typedef {import('../utils').ComponentProp} ComponentProp
+ * @typedef {import('../utils').GroupName} GroupName
+ */
+
+const utils = require('../utils')
+const { isComment } = require('../utils/indent-utils')
```

Comment:
Please use eslint-utils instead.

```suggestion
const { isCommentToken } = require('eslint-utils')
```